frontend/libs/studio-content-library/src/ContentLibrary/LibraryBody/pages/CodeList/hooks/useCodeListEditorTexts.ts

```diff
@@ -0,0 +1,26 @@
+import { useTranslation } from 'react-i18next';
```

Comment:
Har det blitt gjort en vurdering på om vi ønsker å ha en avhengighet til `i18next` fra `content-library`? Det er ikke nødvendigvis dumt, jeg bare lurer på om det er gjennomtenkt., hvis du vet noe om det? Bruker vi dette i denne pakken allerede?

Reply:
Mener jeg hadde en kort dialog på dette med @framitdavid for en stund tilbake og at konklusjonen var at vi kunne dra den inn 🤷